src/main/g8/src/main/scala/com/lightbend/akka/sample/HelloAkka.scala

```diff
@@ -0,0 +1,97 @@
+//#full-example
+package com.lightbend.akka.sample
+
+import akka.actor.{ ActorRef, ActorSystem, Props, Actor, ActorLogging }
+import scala.concurrent.duration._
+import scala.io.StdIn
+
+//#greeter-companion
+//#greeter-messages
+object Greeter {
+  //#greeter-messages
+  def props(message: String, printerActor: ActorRef): Props = Props(new Greeter(message, printerActor))
+  //#greeter-messages
+  case class WhoToGreet(who: String)
+  case object Greet
+}
+//#greeter-messages
+//#greeter-companion
+
+//#greeter-actor
+class Greeter(message: String, printerActor: ActorRef) extends Actor {
+  import Greeter._
+  import Printer._
+
+  var greeting = ""
+
+  def receive = {
+    case WhoToGreet(who) => greeting = s"$message, $who"
```

Comment:
reads slightly better to have a newline after => here, since the next case has that